### PR TITLE
[DIS-870] build: setup backend coverage badge

### PR DESCRIPTION
**Pull request by csanjuanc**
**Goal:** setup backend coverage badge

**How to test:** you can see the change in https://github.com/LinioIT/dynamic-form-bundle/tree/build/DIS-870-add-backend-coverage-to-project

Captura de Pantalla 2021-12-02 a la(s) 10 45 10

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -28,4 +31,15 @@ jobs:
...
-      run: composer phpunit
+      run: vendor/bin/phpunit --coverage-clover coverage.xml
+
+    - name: Generate code coverage badge
+      run: vendor/bin/php-coverage-badger coverage.xml .github/badges/coverage.svg
+
+    - name: Commit code coverage badge
+      uses: EndBug/add-and-commit@v5
+      with:
+        message: "Commit backend coverage badge"
```
Comment (felipegirotti) [inline .github/workflows/php.yml:42]: You should use `[ci skip] feat: Commit backend coverage badge`
To not run the pipeline again,
Reference: 
- GitHub Skip workflow https://docs.github.com/pt/actions/managing-workflow-runs/skipping-workflow-runs
- LinioIT commit messages: https://it-wiki.linio.com/standards/commit/

**Review by felipegirotti — CHANGES_REQUESTED** [review]

File: .github/workflows/php.yml (hunk 1 above)
Comment (tamara-dietz) [reply .github/workflows/php.yml:42]: That's why we wanted the review from someone from Infra, thanks @felipegirotti

Comment (tamara-dietz) [reply .github/workflows/php.yml:42]: That's why we wanted the review from someone from Infra, thanks Felipe.

Comment (csanjuanc) [reply .github/workflows/php.yml:42]: thanks, I will fix it!

**Review by felipegirotti — APPROVED** [review]

**Review by tamara-dietz — APPROVED** [review]

**Pull request merged**